packages/teams-js/src/public/tasks.ts

```diff
@@ -71,7 +71,7 @@ export namespace tasks {
 
   /**
    * @deprecated
-   * As of 2.0.0, please use {@link dialog.url.submit} instead.
+   * As of 2.8.0, please use {@link dialog.url.submit} instead.
```

Comment:
Why the need to bump from 2.0.0 to 2.8.0 in this file?

Reply:
because, as of 2.0.0, url.submit does not exist.